packages/rke2-cilium/package.yaml

```diff
@@ -0,0 +1 @@
+packageVersion: 01
```

Comment:
I'm still a little shaky on the new process, but does this need a URL line in order to specify what the upstream chart that the patch is being applied to?

Reply:
It's getting pulled automatically due to being a requirement, yes, as seen above. If it were not a subchart and we would only like to have a parent chart based of a pulled chart then we would need an url here. The url is only strictly check to perform a download that we don't need in this case:

https://github.com/rancher/rke2-charts/blob/b40561ff50a916ea201798c14904cd7a233c0408/scripts/prepare#L11-L34

But `package.yaml` with `packageVersion` is needed to apply the patch and track different versions of the patch itself.